runtime/foucoco/src/lib.rs

```diff
@@ -339,7 +336,7 @@ impl Contains<RuntimeCall> for BaseFilter {
 			RuntimeCall::Preimage(_) |
 			RuntimeCall::Timestamp(_) |
 			RuntimeCall::Balances(_) |
-			RuntimeCall::Authorship(_) |
+
```

Comment:
`Authorship` does not have a call anymore. See [substrate PR #13216](https://github.com/paritytech/substrate/pull/13216/files#diff-0b4e629e0e921f6705567ff10743bc40706b7045ef47564a9e7de108203be2b3L237)